### PR TITLE
PR - Discord Export Feature

### PR DESCRIPTION
**Pull request by Metalios**
Fixed discord export settings not saving servers etc.
Updated discord export client to accept new responses from the server in regards to the queuing system.

### CODE REVIEW

File: Base/ConfigData.cs [hunk 1]
```diff
@@ -1,9 +1,11 @@
 using System;
+using System.CodeDom;
```
Comment (ImaginaryDevelopment) [inline Base/ConfigData.cs]: why is this here?

File: Hero Designer.sln.DotSettings [hunk 2]
```diff
@@ -1,3 +1,5 @@
 ﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```
Comment (ImaginaryDevelopment) [inline Hero Designer.sln.DotSettings]: I don't think this file should be committed.

File: Hero Designer/clsHook.cs [hunk 3]
```diff
@@ -67,7 +67,9 @@ internal static void DiscordExport()
 
             var discord = (
```
Comment (ImaginaryDevelopment) [inline Hero Designer/clsHook.cs:68]: why a named tuple instead of an anonymous type?

File: Base/ConfigData.cs (hunk 1 above)
Comment (Metalios) [reply Base/ConfigData.cs]: Forgot to remove it from a previous settings save I was working on, technically not only that but also the following can be removed from that file as they are not used.

using System.Linq;
using System.Reflection;
using System.Runtime.CompilerServices;
using System.Runtime.InteropServices;

File: Hero Designer.sln.DotSettings (hunk 2 above)
Comment (Crytilis) [reply Hero Designer.sln.DotSettings]: > This adds the built-in servers to the custom save list, was that intentional?

Yes, as it takes the built in and includes anything added to it, builds a new list then pushes back to the listbox as well as saves in settings.

File: Hero Designer/clsHook.cs (hunk 3 above)
Comment (Metalios) [reply Hero Designer/clsHook.cs:68]: Figured using a tuple would better suit the purpose in this case.

**Review by ImaginaryDevelopment — COMMENTED** [review]
looks good, since I'm required to leave a comment to finish my re-review
